tests/testthat/test-poi-sf.R

```diff
@@ -0,0 +1,57 @@
+context("Simple features support")
```

Comment:
`context()` is no longer recommended. please scrap this line

Reply:
OK
```suggestion
```